wechat_qrcode_test.go

```diff
@@ -0,0 +1,56 @@
+package gocv
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestNewWeChatQRCode(t *testing.T) {
```

Comment:
Hello @dirname 

I am thinking that probably the tests and test model files should be handled conditionally like the test files for Caffe/Tensorflow. 

Please see https://github.com/hybridgroup/gocv/blob/release/dnn_test.go#L98

The reason for this is to keep the size of the repo down to a smaller size. 

Is there a place that the model files you are using for the tests here can be downloaded from, similar to how it is handled here for CircleCI: https://github.com/hybridgroup/gocv/blob/release/.circleci/config.yml#L25-L30 

and then here for Appveyor: https://github.com/hybridgroup/gocv/blob/release/appveyor_build_opencv.cmd#L19

Reply:
@deadprogram Changed, please review again.

Reply:
Thanks for the quick update!